Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1069,6 +1080,8 @@ def __call__(self, message):
         text = self.encode(message)
         if self.buffering:
             self.messages.append(text)
+            if self.debug_logging:
+                demisto.debug(text)
         else:
             demisto.info(text)
```

Comment:
no biiggy, but this might cause some inconsistency where in debug mode the log can be in info level in case `buffering` is off

Reply:
I think it is ok